source/plugin/basics/data/datamanipulator.rst

```diff
@@ -0,0 +1,132 @@
+================
+Datamanipulators
+================
+
+.. note::
+
+    As of writing, the features described in this document are not fully implemented yet.
+
+    If acquiring of a ``DataManipulator`` or a ``DataManipulatorBuilder`` fails, it is probably due to the implementation missing.
```

Comment:
I included this part for the implementation developers, since I figure the implementation will take a while and probably quite a share of people will try to use a `DataManipulator` that may exist but not be fully implemented, so I thought a note like this might prevent some "OMG Stuff is buggy" issues.
You are correct though, if I include this note it must not confuse the reader. Maybe @gabizou can provide the information on how to reliably check if a given DataManipulator is implemented. My guess would be that once a Builder can be found in the registry you're good to go. But that may be too naive.